lib/pipeline.js

```diff
@@ -123,6 +115,35 @@ class PipelineModel extends BaseModel {
     }
 
     /**
+     * Get the first admin username of the pipeline.
+     * @method getFirstAdminName
+     * @return {String}
+     */
+    // TODO: Get the first admin. Validate if it's valid. If not, remove from admins field,
+    //       and retry with the next admin.
+    getFirstAdminName() {
```

Comment:
`getFirstAdminName`, and `getAdmin` should appear in source order before they are used (in `get admin`)